src/schema/v2/object_identification.ts

```diff
@@ -60,6 +60,10 @@ const typeNames = {
   "./filterArtworksConnection": "filterArtworksConnection",
 }
 
+const exportNames = {
```

Comment:
I'm not sure if this was causing a bug in Emission or not (unrelated to the pagination), but noticed the `... on Node` stuff was failing (I had a corresponding spec that was passing locally but must have started failing at some point).

There was a default export that was changed to named (seems to cause weird circular dependency type things). So just added this little override. /shrug